### ISSUE TITLE
Lantern Feedback Form may be Double-Submitting

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#2158]
I've noticed that sometimes we get two messages with the same text.  For example on January 12 2015 we see "The message says it cannot load setting. It message appears when the system is booting." from the same Lantern instance twice.

**Comment by myleshorton** [comment]
I believe this is more of an issue of the UI not giving feedback when a form is submitted, as otherwise we'd see double submissions every time which we definitely don't see. Closing for now.

**Comment by oxtoacart** [comment]
> UI not giving feedback when a form is submitted

Isn't that an issue?  Why close the ticket?  It's already a low priority, not short-term item.

**Comment by myleshorton** [comment]
> Isn't that an issue? Why close the ticket? It's already a low priority, not short-term item.

Yeah, just a different issue/ticket. Actually, though, the feedback form just disappears after submitting, so not sure what's going on.

**Comment by oxtoacart** [comment]
Reopening for investigation.

**Comment by atavism** [comment]
> I believe this is more of an issue of the UI not giving feedback when a form is submitted, as otherwise we'd see double submissions every time which we definitely don't see. Closing for now.

When do you see this? I tested on 0.0.12, and I'm able to submit the contact form fine. I get a notice afterwards that my message was successfully sent. I also checked Loggly, and the message didn't show up multiple times.

**Comment by oxtoacart** [comment]
@myleshorton @atavism 

Thanks very much for looking into this!  As mentioned, I just opened this bug because I've observed what look like double submissions in Loggly from time to time.  I have no idea whether they're even accidental, and if so whether it's caused by the UI or if there's some bug in the logging code.  It's not really a big deal, hence the P3 designation.  I wouldn't worry about it for now :)
